packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -79,7 +79,7 @@ class WebEntrypointTarget extends Target {
     final String importedEntrypoint = packageConfig.toPackageUri(importUri)?.toString()
       ?? importUri.toString();
 
-    await injectBuildTimePluginFiles(flutterProject, webPlatform: true, destination: environment.buildDir);
+    await injectBuildTimePluginFilesForWebPlatform(flutterProject, destination: environment.buildDir);
```

Comment:
Nit: since we are already creating a diff here, let's format this:

```suggestion
    await injectBuildTimePluginFilesForWebPlatform(
      flutterProject,
      destination: environment.buildDir,
    );
```